Comparing source compatibility of /home/jenkins/.m2/repository/org/apache/jspwiki/jspwiki-markdown/2.11.0.M5/jspwiki-markdown-2.11.0.M5.jar against /home/jenkins/.m2/repository/org/apache/jspwiki/jspwiki-markdown/2.10.5/jspwiki-markdown-2.10.5.jar
WARNING: You are using the option '--ignore-missing-classes', i.e. superclasses and interfaces that could not be found on the classpath are ignored. Hence changes caused by these superclasses and interfaces are not reflected in the output.
***  MODIFIED CLASS: PUBLIC org.apache.wiki.markdown.extensions.jspwikilinks.attributeprovider.ExternalLinkAttributeProviderState  (not serializable)
	***! CLASS FILE FORMAT VERSION: 52.0 <- 51.0
	===! UNCHANGED INTERFACE: org.apache.wiki.markdown.extensions.jspwikilinks.attributeprovider.NodeAttributeProviderState
	===  UNCHANGED CONSTRUCTOR: PUBLIC ExternalLinkAttributeProviderState(org.apache.wiki.WikiContext, boolean)
	===  UNCHANGED METHOD: PUBLIC void setAttributes(com.vladsch.flexmark.util.html.Attributes, org.apache.wiki.markdown.nodes.JSPWikiLink)
***  MODIFIED CLASS: PUBLIC org.apache.wiki.markdown.extensions.jspwikilinks.attributeprovider.ImageLinkAttributeProviderState  (not serializable)
	***! CLASS FILE FORMAT VERSION: 52.0 <- 51.0
	===! UNCHANGED INTERFACE: org.apache.wiki.markdown.extensions.jspwikilinks.attributeprovider.NodeAttributeProviderState
	===  UNCHANGED CONSTRUCTOR: PUBLIC ImageLinkAttributeProviderState(org.apache.wiki.WikiContext, java.lang.String, boolean)
	===  UNCHANGED METHOD: PUBLIC void setAttributes(com.vladsch.flexmark.util.html.Attributes, org.apache.wiki.markdown.nodes.JSPWikiLink)
***  MODIFIED CLASS: PUBLIC org.apache.wiki.markdown.extensions.jspwikilinks.attributeprovider.InterWikiLinkAttributeProviderState  (not serializable)
	***! CLASS FILE FORMAT VERSION: 52.0 <- 51.0
	===! UNCHANGED INTERFACE: org.apache.wiki.markdown.extensions.jspwikilinks.attributeprovider.NodeAttributeProviderState
	===  UNCHANGED CONSTRUCTOR: PUBLIC InterWikiLinkAttributeProviderState(org.apache.wiki.WikiContext, boolean)
	===  UNCHANGED METHOD: PUBLIC void setAttributes(com.vladsch.flexmark.util.html.Attributes, org.apache.wiki.markdown.nodes.JSPWikiLink)
***! MODIFIED CLASS: PUBLIC org.apache.wiki.markdown.extensions.jspwikilinks.attributeprovider.JSPWikiLinkAttributeProvider  (not serializable)
	***! CLASS FILE FORMAT VERSION: 52.0 <- 51.0
	===  UNCHANGED FIELD: PROTECTED FINAL org.apache.wiki.parser.LinkParsingOperations linkOperations
	===  UNCHANGED FIELD: PROTECTED FINAL org.apache.wiki.WikiContext wikiContext
	===  UNCHANGED CONSTRUCTOR: PUBLIC JSPWikiLinkAttributeProvider(org.apache.wiki.WikiContext)
	---! REMOVED METHOD: PUBLIC(-) void setAttributes(com.vladsch.flexmark.ast.Node, com.vladsch.flexmark.html.renderer.AttributablePart, com.vladsch.flexmark.util.html.Attributes)
	+++  NEW METHOD: PUBLIC(+) void setAttributes(com.vladsch.flexmark.util.ast.Node, com.vladsch.flexmark.html.renderer.AttributablePart, com.vladsch.flexmark.util.html.Attributes)
***! MODIFIED CLASS: PUBLIC org.apache.wiki.markdown.extensions.jspwikilinks.attributeprovider.JSPWikiLinkAttributeProviderFactory  (not serializable)
	***! CLASS FILE FORMAT VERSION: 52.0 <- 51.0
	===  UNCHANGED CONSTRUCTOR: PUBLIC JSPWikiLinkAttributeProviderFactory(org.apache.wiki.WikiContext)
	+++  NEW METHOD: PUBLIC(+) com.vladsch.flexmark.html.AttributeProvider apply(com.vladsch.flexmark.html.renderer.LinkResolverContext)
	---! REMOVED METHOD: PUBLIC(-) com.vladsch.flexmark.html.AttributeProvider create(com.vladsch.flexmark.html.renderer.LinkResolverContext)
***  MODIFIED CLASS: PUBLIC org.apache.wiki.markdown.extensions.jspwikilinks.attributeprovider.LocalEditLinkAttributeProviderState  (not serializable)
	***! CLASS FILE FORMAT VERSION: 52.0 <- 51.0
	===! UNCHANGED INTERFACE: org.apache.wiki.markdown.extensions.jspwikilinks.attributeprovider.NodeAttributeProviderState
	===  UNCHANGED CONSTRUCTOR: PUBLIC LocalEditLinkAttributeProviderState(org.apache.wiki.WikiContext, java.lang.String)
	===  UNCHANGED METHOD: PUBLIC void setAttributes(com.vladsch.flexmark.util.html.Attributes, org.apache.wiki.markdown.nodes.JSPWikiLink)
***  MODIFIED CLASS: PUBLIC org.apache.wiki.markdown.extensions.jspwikilinks.attributeprovider.LocalFootnoteLinkAttributeProviderState  (not serializable)
	***! CLASS FILE FORMAT VERSION: 52.0 <- 51.0
	===! UNCHANGED INTERFACE: org.apache.wiki.markdown.extensions.jspwikilinks.attributeprovider.NodeAttributeProviderState
	===  UNCHANGED CONSTRUCTOR: PUBLIC LocalFootnoteLinkAttributeProviderState(org.apache.wiki.WikiContext)
	===  UNCHANGED METHOD: PUBLIC void setAttributes(com.vladsch.flexmark.util.html.Attributes, org.apache.wiki.markdown.nodes.JSPWikiLink)
***  MODIFIED CLASS: PUBLIC org.apache.wiki.markdown.extensions.jspwikilinks.attributeprovider.LocalFootnoteRefLinkAttributeProviderState  (not serializable)
	***! CLASS FILE FORMAT VERSION: 52.0 <- 51.0
	===! UNCHANGED INTERFACE: org.apache.wiki.markdown.extensions.jspwikilinks.attributeprovider.NodeAttributeProviderState
	===  UNCHANGED CONSTRUCTOR: PUBLIC LocalFootnoteRefLinkAttributeProviderState(org.apache.wiki.WikiContext)
	===  UNCHANGED METHOD: PUBLIC void setAttributes(com.vladsch.flexmark.util.html.Attributes, org.apache.wiki.markdown.nodes.JSPWikiLink)
***  MODIFIED CLASS: PUBLIC org.apache.wiki.markdown.extensions.jspwikilinks.attributeprovider.LocalLinkAttributeProviderState  (not serializable)
	***! CLASS FILE FORMAT VERSION: 52.0 <- 51.0
	===! UNCHANGED INTERFACE: org.apache.wiki.markdown.extensions.jspwikilinks.attributeprovider.NodeAttributeProviderState
	===  UNCHANGED CONSTRUCTOR: PUBLIC LocalLinkAttributeProviderState(org.apache.wiki.WikiContext, boolean)
	===  UNCHANGED METHOD: PUBLIC void setAttributes(com.vladsch.flexmark.util.html.Attributes, org.apache.wiki.markdown.nodes.JSPWikiLink)
***  MODIFIED CLASS: PUBLIC org.apache.wiki.markdown.extensions.jspwikilinks.attributeprovider.LocalReadLinkAttributeProviderState  (not serializable)
	***! CLASS FILE FORMAT VERSION: 52.0 <- 51.0
	===! UNCHANGED INTERFACE: org.apache.wiki.markdown.extensions.jspwikilinks.attributeprovider.NodeAttributeProviderState
	===  UNCHANGED CONSTRUCTOR: PUBLIC LocalReadLinkAttributeProviderState(org.apache.wiki.WikiContext)
	===  UNCHANGED METHOD: PUBLIC void setAttributes(com.vladsch.flexmark.util.html.Attributes, org.apache.wiki.markdown.nodes.JSPWikiLink)
***! MODIFIED INTERFACE: PUBLIC ABSTRACT org.apache.wiki.markdown.extensions.jspwikilinks.attributeprovider.NodeAttributeProviderState  (not serializable)
	***! CLASS FILE FORMAT VERSION: 52.0 <- 51.0
	---! REMOVED METHOD: PUBLIC(-) ABSTRACT(-) void setAttributes(com.vladsch.flexmark.util.html.Attributes, com.vladsch.flexmark.ast.Node)
	+++* NEW METHOD: PUBLIC(+) ABSTRACT(+) void setAttributes(com.vladsch.flexmark.util.html.Attributes, com.vladsch.flexmark.util.ast.Node)
***  MODIFIED CLASS: PUBLIC org.apache.wiki.markdown.extensions.jspwikilinks.attributeprovider.WysiwygEditingAttributeProviderState  (not serializable)
	***! CLASS FILE FORMAT VERSION: 52.0 <- 51.0
	===! UNCHANGED INTERFACE: org.apache.wiki.markdown.extensions.jspwikilinks.attributeprovider.NodeAttributeProviderState
	===  UNCHANGED CONSTRUCTOR: PUBLIC WysiwygEditingAttributeProviderState(org.apache.wiki.WikiContext)
	===  UNCHANGED METHOD: PUBLIC void setAttributes(com.vladsch.flexmark.util.html.Attributes, org.apache.wiki.markdown.nodes.JSPWikiLink)
***! MODIFIED CLASS: PUBLIC org.apache.wiki.markdown.extensions.jspwikilinks.postprocessor.AccessRuleLinkNodePostProcessorState  (not serializable)
	***! CLASS FILE FORMAT VERSION: 52.0 <- 51.0
	===! UNCHANGED INTERFACE: org.apache.wiki.markdown.extensions.jspwikilinks.postprocessor.NodePostProcessorState
	===  UNCHANGED CONSTRUCTOR: PUBLIC AccessRuleLinkNodePostProcessorState(org.apache.wiki.WikiContext)
	---! REMOVED METHOD: PUBLIC(-) void process(com.vladsch.flexmark.util.NodeTracker, org.apache.wiki.markdown.nodes.JSPWikiLink)
	+++  NEW METHOD: PUBLIC(+) void process(com.vladsch.flexmark.util.ast.NodeTracker, org.apache.wiki.markdown.nodes.JSPWikiLink)
***! MODIFIED CLASS: PUBLIC org.apache.wiki.markdown.extensions.jspwikilinks.postprocessor.ExternalLinkNodePostProcessorState  (not serializable)
	***! CLASS FILE FORMAT VERSION: 52.0 <- 51.0
	===! UNCHANGED INTERFACE: org.apache.wiki.markdown.extensions.jspwikilinks.postprocessor.NodePostProcessorState
	===  UNCHANGED CONSTRUCTOR: PUBLIC ExternalLinkNodePostProcessorState(org.apache.wiki.WikiContext)
	---! REMOVED METHOD: PUBLIC(-) void process(com.vladsch.flexmark.util.NodeTracker, org.apache.wiki.markdown.nodes.JSPWikiLink)
	+++  NEW METHOD: PUBLIC(+) void process(com.vladsch.flexmark.util.ast.NodeTracker, org.apache.wiki.markdown.nodes.JSPWikiLink)
***! MODIFIED CLASS: PUBLIC org.apache.wiki.markdown.extensions.jspwikilinks.postprocessor.ImageLinkNodePostProcessorState  (not serializable)
	***! CLASS FILE FORMAT VERSION: 52.0 <- 51.0
	===! UNCHANGED INTERFACE: org.apache.wiki.markdown.extensions.jspwikilinks.postprocessor.NodePostProcessorState
	===  UNCHANGED CONSTRUCTOR: PUBLIC ImageLinkNodePostProcessorState(org.apache.wiki.WikiContext, java.lang.String, boolean)
	---! REMOVED METHOD: PUBLIC(-) void process(com.vladsch.flexmark.util.NodeTracker, org.apache.wiki.markdown.nodes.JSPWikiLink)
	+++  NEW METHOD: PUBLIC(+) void process(com.vladsch.flexmark.util.ast.NodeTracker, org.apache.wiki.markdown.nodes.JSPWikiLink)
***! MODIFIED CLASS: PUBLIC org.apache.wiki.markdown.extensions.jspwikilinks.postprocessor.InterWikiLinkNodePostProcessorState  (not serializable)
	***! CLASS FILE FORMAT VERSION: 52.0 <- 51.0
	===! UNCHANGED INTERFACE: org.apache.wiki.markdown.extensions.jspwikilinks.postprocessor.NodePostProcessorState
	---! REMOVED CONSTRUCTOR: PUBLIC(-) InterWikiLinkNodePostProcessorState(org.apache.wiki.WikiContext, com.vladsch.flexmark.ast.Document)
	+++  NEW CONSTRUCTOR: PUBLIC(+) InterWikiLinkNodePostProcessorState(org.apache.wiki.WikiContext, com.vladsch.flexmark.util.ast.Document)
	---! REMOVED METHOD: PUBLIC(-) void process(com.vladsch.flexmark.util.NodeTracker, org.apache.wiki.markdown.nodes.JSPWikiLink)
	+++  NEW METHOD: PUBLIC(+) void process(com.vladsch.flexmark.util.ast.NodeTracker, org.apache.wiki.markdown.nodes.JSPWikiLink)
***! MODIFIED CLASS: PUBLIC org.apache.wiki.markdown.extensions.jspwikilinks.postprocessor.JSPWikiLinkNodePostProcessor  (not serializable)
	***! CLASS FILE FORMAT VERSION: 52.0 <- 51.0
	===  UNCHANGED FIELD: PROTECTED FINAL org.apache.wiki.WikiContext m_context
	***! MODIFIED FIELD: PROTECTED FINAL com.vladsch.flexmark.util.ast.Document (<- com.vladsch.flexmark.ast.Document) document
	===  UNCHANGED FIELD: PROTECTED FINAL org.apache.wiki.parser.LinkParsingOperations linkOperations
	===  UNCHANGED FIELD: PROTECTED boolean m_useOutlinkImage
	---! REMOVED CONSTRUCTOR: PUBLIC(-) JSPWikiLinkNodePostProcessor(org.apache.wiki.WikiContext, com.vladsch.flexmark.ast.Document)
	+++  NEW CONSTRUCTOR: PUBLIC(+) JSPWikiLinkNodePostProcessor(org.apache.wiki.WikiContext, com.vladsch.flexmark.util.ast.Document)
	---! REMOVED METHOD: PUBLIC(-) void process(com.vladsch.flexmark.util.NodeTracker, com.vladsch.flexmark.ast.Node)
	+++  NEW METHOD: PUBLIC(+) void process(com.vladsch.flexmark.util.ast.NodeTracker, com.vladsch.flexmark.util.ast.Node)
***! MODIFIED CLASS: PUBLIC org.apache.wiki.markdown.extensions.jspwikilinks.postprocessor.JSPWikiNodePostProcessorFactory  (not serializable)
	***! CLASS FILE FORMAT VERSION: 52.0 <- 51.0
	---! REMOVED CONSTRUCTOR: PUBLIC(-) JSPWikiNodePostProcessorFactory(org.apache.wiki.WikiContext, com.vladsch.flexmark.util.options.DataHolder)
	+++  NEW CONSTRUCTOR: PUBLIC(+) JSPWikiNodePostProcessorFactory(org.apache.wiki.WikiContext, com.vladsch.flexmark.util.data.DataHolder)
	+++  NEW METHOD: PUBLIC(+) com.vladsch.flexmark.parser.block.NodePostProcessor apply(com.vladsch.flexmark.util.ast.Document)
	---! REMOVED METHOD: PUBLIC(-) com.vladsch.flexmark.parser.block.NodePostProcessor create(com.vladsch.flexmark.ast.Document)
***! MODIFIED CLASS: PUBLIC org.apache.wiki.markdown.extensions.jspwikilinks.postprocessor.LocalFootnoteLinkNodePostProcessorState  (not serializable)
	***! CLASS FILE FORMAT VERSION: 52.0 <- 51.0
	===! UNCHANGED INTERFACE: org.apache.wiki.markdown.extensions.jspwikilinks.postprocessor.NodePostProcessorState
	===  UNCHANGED CONSTRUCTOR: PUBLIC LocalFootnoteLinkNodePostProcessorState(org.apache.wiki.WikiContext)
	---! REMOVED METHOD: PUBLIC(-) void process(com.vladsch.flexmark.util.NodeTracker, org.apache.wiki.markdown.nodes.JSPWikiLink)
	+++  NEW METHOD: PUBLIC(+) void process(com.vladsch.flexmark.util.ast.NodeTracker, org.apache.wiki.markdown.nodes.JSPWikiLink)
***! MODIFIED CLASS: PUBLIC org.apache.wiki.markdown.extensions.jspwikilinks.postprocessor.LocalFootnoteRefLinkNodePostProcessorState  (not serializable)
	***! CLASS FILE FORMAT VERSION: 52.0 <- 51.0
	===! UNCHANGED INTERFACE: org.apache.wiki.markdown.extensions.jspwikilinks.postprocessor.NodePostProcessorState
	===  UNCHANGED CONSTRUCTOR: PUBLIC LocalFootnoteRefLinkNodePostProcessorState(org.apache.wiki.WikiContext)
	---! REMOVED METHOD: PUBLIC(-) void process(com.vladsch.flexmark.util.NodeTracker, org.apache.wiki.markdown.nodes.JSPWikiLink)
	+++  NEW METHOD: PUBLIC(+) void process(com.vladsch.flexmark.util.ast.NodeTracker, org.apache.wiki.markdown.nodes.JSPWikiLink)
***! MODIFIED CLASS: PUBLIC org.apache.wiki.markdown.extensions.jspwikilinks.postprocessor.LocalLinkNodePostProcessorState  (not serializable)
	***! CLASS FILE FORMAT VERSION: 52.0 <- 51.0
	===! UNCHANGED INTERFACE: org.apache.wiki.markdown.extensions.jspwikilinks.postprocessor.NodePostProcessorState
	===  UNCHANGED CONSTRUCTOR: PUBLIC LocalLinkNodePostProcessorState(org.apache.wiki.WikiContext)
	---! REMOVED METHOD: PUBLIC(-) void process(com.vladsch.flexmark.util.NodeTracker, org.apache.wiki.markdown.nodes.JSPWikiLink)
	+++  NEW METHOD: PUBLIC(+) void process(com.vladsch.flexmark.util.ast.NodeTracker, org.apache.wiki.markdown.nodes.JSPWikiLink)
***! MODIFIED CLASS: PUBLIC org.apache.wiki.markdown.extensions.jspwikilinks.postprocessor.MetadataLinkNodePostProcessorState  (not serializable)
	***! CLASS FILE FORMAT VERSION: 52.0 <- 51.0
	===! UNCHANGED INTERFACE: org.apache.wiki.markdown.extensions.jspwikilinks.postprocessor.NodePostProcessorState
	===  UNCHANGED CONSTRUCTOR: PUBLIC MetadataLinkNodePostProcessorState(org.apache.wiki.WikiContext)
	---! REMOVED METHOD: PUBLIC(-) void process(com.vladsch.flexmark.util.NodeTracker, org.apache.wiki.markdown.nodes.JSPWikiLink)
	+++  NEW METHOD: PUBLIC(+) void process(com.vladsch.flexmark.util.ast.NodeTracker, org.apache.wiki.markdown.nodes.JSPWikiLink)
***! MODIFIED INTERFACE: PUBLIC ABSTRACT org.apache.wiki.markdown.extensions.jspwikilinks.postprocessor.NodePostProcessorState  (not serializable)
	***! CLASS FILE FORMAT VERSION: 52.0 <- 51.0
	---! REMOVED METHOD: PUBLIC(-) ABSTRACT(-) void process(com.vladsch.flexmark.util.NodeTracker, com.vladsch.flexmark.ast.Node)
	+++* NEW METHOD: PUBLIC(+) ABSTRACT(+) void process(com.vladsch.flexmark.util.ast.NodeTracker, com.vladsch.flexmark.util.ast.Node)
***! MODIFIED CLASS: PUBLIC org.apache.wiki.markdown.extensions.jspwikilinks.postprocessor.PluginLinkNodePostProcessorState  (not serializable)
	***! CLASS FILE FORMAT VERSION: 52.0 <- 51.0
	===! UNCHANGED INTERFACE: org.apache.wiki.markdown.extensions.jspwikilinks.postprocessor.NodePostProcessorState
	===  UNCHANGED CONSTRUCTOR: PUBLIC PluginLinkNodePostProcessorState(org.apache.wiki.WikiContext)
	---! REMOVED METHOD: PUBLIC(-) void process(com.vladsch.flexmark.util.NodeTracker, org.apache.wiki.markdown.nodes.JSPWikiLink)
	+++  NEW METHOD: PUBLIC(+) void process(com.vladsch.flexmark.util.ast.NodeTracker, org.apache.wiki.markdown.nodes.JSPWikiLink)
***! MODIFIED CLASS: PUBLIC org.apache.wiki.markdown.extensions.jspwikilinks.postprocessor.VariableLinkNodePostProcessorState  (not serializable)
	***! CLASS FILE FORMAT VERSION: 52.0 <- 51.0
	===! UNCHANGED INTERFACE: org.apache.wiki.markdown.extensions.jspwikilinks.postprocessor.NodePostProcessorState
	===  UNCHANGED CONSTRUCTOR: PUBLIC VariableLinkNodePostProcessorState(org.apache.wiki.WikiContext)
	---! REMOVED METHOD: PUBLIC(-) void process(com.vladsch.flexmark.util.NodeTracker, org.apache.wiki.markdown.nodes.JSPWikiLink)
	+++  NEW METHOD: PUBLIC(+) void process(com.vladsch.flexmark.util.ast.NodeTracker, org.apache.wiki.markdown.nodes.JSPWikiLink)
***! MODIFIED CLASS: PUBLIC org.apache.wiki.markdown.MarkdownForJSPWikiExtension  (not serializable)
	***! CLASS FILE FORMAT VERSION: 52.0 <- 51.0
	===  UNCHANGED CONSTRUCTOR: PUBLIC MarkdownForJSPWikiExtension(org.apache.wiki.WikiContext)
	===  UNCHANGED METHOD: PUBLIC void extend(com.vladsch.flexmark.html.HtmlRenderer$Builder, java.lang.String)
	===  UNCHANGED METHOD: PUBLIC void extend(com.vladsch.flexmark.parser.Parser$Builder)
	---! REMOVED METHOD: PUBLIC(-) void parserOptions(com.vladsch.flexmark.util.options.MutableDataHolder)
	+++  NEW METHOD: PUBLIC(+) void parserOptions(com.vladsch.flexmark.util.data.MutableDataHolder)
	---! REMOVED METHOD: PUBLIC(-) void rendererOptions(com.vladsch.flexmark.util.options.MutableDataHolder)
	+++  NEW METHOD: PUBLIC(+) void rendererOptions(com.vladsch.flexmark.util.data.MutableDataHolder)
===  UNCHANGED CLASS: PUBLIC org.apache.wiki.markdown.nodes.JSPWikiLink  (not serializable)
	***! CLASS FILE FORMAT VERSION: 52.0 <- 51.0
	===  UNCHANGED CONSTRUCTOR: PUBLIC JSPWikiLink(com.vladsch.flexmark.ast.Link)
	===  UNCHANGED METHOD: PUBLIC java.lang.String getWikiLink()
	===  UNCHANGED METHOD: PUBLIC boolean hasRef()
	===  UNCHANGED METHOD: PUBLIC java.lang.String toStringAttributes()
===  UNCHANGED CLASS: PUBLIC org.apache.wiki.markdown.renderer.JSPWikiLinkRenderer  (not serializable)
	***! CLASS FILE FORMAT VERSION: 52.0 <- 51.0
	===  UNCHANGED CONSTRUCTOR: PUBLIC JSPWikiLinkRenderer()
	===  UNCHANGED METHOD: PUBLIC java.util.Set getNodeRenderingHandlers()
***! MODIFIED CLASS: PUBLIC org.apache.wiki.markdown.renderer.JSPWikiNodeRendererFactory  (not serializable)
	***! CLASS FILE FORMAT VERSION: 52.0 <- 51.0
	===  UNCHANGED CONSTRUCTOR: PUBLIC JSPWikiNodeRendererFactory(org.apache.wiki.WikiContext)
	+++  NEW METHOD: PUBLIC(+) com.vladsch.flexmark.html.renderer.NodeRenderer apply(com.vladsch.flexmark.util.data.DataHolder)
	---! REMOVED METHOD: PUBLIC(-) com.vladsch.flexmark.html.renderer.NodeRenderer create(com.vladsch.flexmark.util.options.DataHolder)
	===  UNCHANGED METHOD: PUBLIC java.util.Set getDelegates()
***! MODIFIED CLASS: PUBLIC org.apache.wiki.parser.markdown.MarkdownDocument  (not serializable)
	***! CLASS FILE FORMAT VERSION: 52.0 <- 51.0
	---! REMOVED CONSTRUCTOR: PUBLIC(-) MarkdownDocument(org.apache.wiki.WikiPage, com.vladsch.flexmark.ast.Node)
	+++  NEW CONSTRUCTOR: PUBLIC(+) MarkdownDocument(org.apache.wiki.WikiPage, com.vladsch.flexmark.util.ast.Node)
	***! MODIFIED METHOD: PUBLIC com.vladsch.flexmark.util.ast.Node (<-com.vladsch.flexmark.ast.Node) getMarkdownNode()
	***! MODIFIED METHOD: PUBLIC STATIC com.vladsch.flexmark.util.data.MutableDataSet (<-com.vladsch.flexmark.util.options.MutableDataSet) options(org.apache.wiki.WikiContext)
===  UNCHANGED CLASS: PUBLIC org.apache.wiki.parser.markdown.MarkdownParser  (not serializable)
	***! CLASS FILE FORMAT VERSION: 52.0 <- 51.0
	===  UNCHANGED CONSTRUCTOR: PUBLIC MarkdownParser(org.apache.wiki.WikiContext, java.io.Reader)
	===  UNCHANGED METHOD: PUBLIC org.apache.wiki.parser.WikiDocument parse()
		===  UNCHANGED EXCEPTION: java.io.IOException
===  UNCHANGED CLASS: PUBLIC org.apache.wiki.render.markdown.MarkdownRenderer  (not serializable)
	***! CLASS FILE FORMAT VERSION: 52.0 <- 51.0
	===  UNCHANGED CONSTRUCTOR: PUBLIC MarkdownRenderer(org.apache.wiki.WikiContext, org.apache.wiki.parser.WikiDocument)
	===  UNCHANGED METHOD: PUBLIC java.lang.String getString()
		===  UNCHANGED EXCEPTION: java.io.IOException
